hack/quickstart/init-master.sh

```diff
@@ -57,11 +69,20 @@ function init_master_node() {
     systemctl enable kubelet; sudo systemctl start kubelet
 
     # Start bootkube to launch a self-hosted cluster
-    /usr/bin/rkt run \
-        --volume home,kind=host,source=/home/core \
-        --mount volume=home,target=/core \
-        --net=host ${BOOTKUBE_REPO}:${BOOTKUBE_VERSION} --exec \
-        /bootkube -- start --asset-dir=/core/assets
+    if [ "$SELF_HOST_ETCD" = true ] ; then
+        /usr/bin/rkt run \
+            --volume home,kind=host,source=/home/core \
+            --mount volume=home,target=/core \
+            --net=host ${BOOTKUBE_REPO}:${BOOTKUBE_VERSION} --exec \
+            /bootkube -- start --asset-dir=/core/assets \
+            --etcd-server=http://${COREOS_PRIVATE_IPV4}:12379 --experimental-self-hosted-etcd
+    else
+        /usr/bin/rkt run \
+            --volume home,kind=host,source=/home/core \
+            --mount volume=home,target=/core \
+            --net=host ${BOOTKUBE_REPO}:${BOOTKUBE_VERSION} --exec \
+            /bootkube -- start --asset-dir=/core/assets
+    fi
```

Comment:
Same comment as above - all the flags could be set in one place and we don't need these if/then blocks